resources/service.rb

```diff
@@ -0,0 +1,45 @@
+# Copyright:: Copyright (c) 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+resource_name :hab_service
+provides :hab_service
+
+property :service_name, String, name_property: true
+property :sup_options, [Array, String]
```

Comment:
should coerce this into an Array.  it doesn't matter now, but it avoids a breaking change later if it ever does matter, and it usually winds up mattering.